cardano-config/src/Cardano/Config/LedgerQueries.hs

```diff
@@ -0,0 +1,42 @@
+{-# LANGUAGE FlexibleInstances #-}
+
+module Cardano.Config.LedgerQueries
+  (LedgerQueries(..))
+where
+
+import           Prelude (Int, error, (.))
+
+import qualified Data.Map as Map
+
+import           Ouroboros.Consensus.Ledger.Abstract
+import           Byron.Spec.Ledger.Core (Relation(..))
+
+import qualified Cardano.Chain.Block as Byron
+import qualified Cardano.Chain.UTxO as Byron
+import qualified Ouroboros.Consensus.Byron.Ledger.Block as Byron
+import qualified Ouroboros.Consensus.Byron.Ledger.Ledger as Byron
+
+import qualified Ouroboros.Consensus.Shelley.Ledger as Shelley
+import qualified Shelley.Spec.Ledger.LedgerState as Shelley
+import qualified Shelley.Spec.Ledger.UTxO as Shelley
+
+import qualified Ouroboros.Consensus.Cardano as Cardano
+
+import qualified Ouroboros.Consensus.Mock.Ledger as Mock
+
+
+class LedgerQueries blk where
+  ledgerUtxoSize :: LedgerState blk -> Int
```

Comment:
I have seen `UTxO` (vs. `Utxo`) in a few places, although `Utxo` follows Haskell's capitalisation rules better.

Reply:
Yes, that was one the unstated questions I had, actually, thank you!

Reply:
So, Thomas, if you think we should change it, I'd need a slightly less subtle suggestion. : -)

Reply:
For the record, ledger code uses:

- `UTXO`
- `UTxO`
- `Utxo`
- `utxo`